cmd/info/info.go

```diff
@@ -0,0 +1,69 @@
+package info
+
+import (
+	"github.com/CircleCI-Public/circleci-cli/api/info"
+	"github.com/CircleCI-Public/circleci-cli/settings"
+	"github.com/olekukonko/tablewriter"
+
+	"github.com/spf13/cobra"
+)
+
+//infoOptions info command options
+type infoOptions struct {
+	cfg       *settings.Config
+	validator validator
+}
+
+type validator func(cmd *cobra.Command, args []string) error
+
+// NewInfoCommand information cobra command creation
+func NewInfoCommand(config *settings.Config, preRunE validator) *cobra.Command {
+	client, _ := info.NewInfoClient(*config)
+
+	opts := infoOptions{
+		cfg: config,
+	}
+	infoCommand := &cobra.Command{
+		Use:   "info",
+		Short: "Check information associated to your user account.",
+	}
+	orgInfoCmd := orgInfoCommand(client, opts)
+	infoCommand.AddCommand(orgInfoCmd)
+
+	return infoCommand
+}
+
+//orgInfoCommand organization information subcommand cobra command creation
+func orgInfoCommand(client info.InfoClient, opts infoOptions) *cobra.Command {
+	return &cobra.Command{
+		Use:     "org",
+		Short:   "View your Organizations' information",
+		Long:    `View your Organizations' names and ids.`,
+		PreRunE: opts.validator,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			return getOrgInformation(cmd, client, opts)
+		},
+		Annotations: make(map[string]string),
+		Example:     `circleci info org`,
+	}
+}
+
+//getOrgInformation gets all of the users organizations' information
+func getOrgInformation(cmd *cobra.Command, client info.InfoClient, opts infoOptions) error {
```

Comment:
the linter was giving me a warning locally since `opts` is unused, it suggested swapping out the unused var to `_`
```suggestion
func getOrgInformation(cmd *cobra.Command, client info.InfoClient, _ infoOptions) error {
```